Documentation/ApiOverview/Logging/Writers/Index.rst

```diff
@@ -167,6 +167,48 @@ Only in the case that all registered writers fail, the log entry plus additional
 will be added to the configured fallback logger (which defaults to
 the :ref:`PhpErrorLog <logging-writers-php>` writer).
 
+.. _logging-writers-usage:
+
+Usage in Custom Class
+---------------------
+
+All log writers can be used in your own classes. You can initialize the loggers in the __construct method of your class :code:`\MyDomain\MyExtension\MyFolder\MyClass`.
+
+.. code-block:: php
+
+    namespace MyDomain\MyExtension\MyFolder;
+
+    use TYPO3\CMS\Core\Utility\GeneralUtility;
+    use TYPO3\CMS\Core\Log\LogManager;
+
+    class MyClass {
+        private $logger;
+
+        public function __construct()
+        {
+            $this->logger = GeneralUtility::makeInstance(LogManager::class)->getLogger(__CLASS__);
+        }
+    }
+
+
+The file ext_localconf.php must have the needed configuration entries for the class from above:
+
+.. code-block:: php
+
+   // Add example configuration for the logging API
```

Comment:
I'm not sure if it makes sense to duplicate the following page:

https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ApiOverview/Logging/Configuration/Index.html

Reply:
The configuration example is helpful. But the configuration could also be shortened.